core/src/main/java/io/seata/core/store/db/LogStoreSqls.java

```diff
@@ -0,0 +1,323 @@
+
+    /**
+     * The constant ALL_BRANCH_COLUMNS.
+     */
+    protected static final String ALL_BRANCH_COLUMNS = "xid, transaction_id, branch_id, resource_group_id, resource_id, lock_key, branch_type, status, client_id, application_data, gmt_create, gmt_modified ";
+
+    /**
+     * The constant INSERT_GLOBAL_TRANSACTION_MYSQL.
+     */
+    public static final String INSERT_GLOBAL_TRANSACTION_MYSQL = "insert into " + GLOBAL_TABLE_PLACEHOLD + "("+ ALL_GLOBAL_COLUMNS +")" +
+            "values(?, ?, ?, ?, ?, ?, ?, ?, ?, now(), now()) ";
+
+    /**
+     * The constant INSERT_GLOBAL_TRANSACTION_ORACLE.
+     */
+    public static final String INSERT_GLOBAL_TRANSACTION_ORACLE = "insert into " + GLOBAL_TABLE_PLACEHOLD + "("+ ALL_GLOBAL_COLUMNS +")" +
+            "values(?, ?, ?, ?, ?, ?, ?, ?, ?, sysdate, sysdate) ";
+
+    /**
+     * The constant UPDATE_GLOBAL_TRANSACTION_STATUS_MYSQL.
+     */
+    public static final String UPDATE_GLOBAL_TRANSACTION_STATUS_MYSQL = "update "+ GLOBAL_TABLE_PLACEHOLD + " set status = ?, gmt_modified = now() where xid = ?";
+
+    /**
+     * The constant UPDATE_GLOBAL_TRANSACTION_STATUS_ORACLE.
+     */
+    public static final String UPDATE_GLOBAL_TRANSACTION_STATUS_ORACLE = "update "+ GLOBAL_TABLE_PLACEHOLD + " set status = ?, gmt_modified = sysdate where xid = ?";
+
+    /**
+     * The constant DELETE_GLOBAL_TRANSACTION.
+     */
+    public static final String DELETE_GLOBAL_TRANSACTION = "delete from " + GLOBAL_TABLE_PLACEHOLD + " where xid = ?";
+
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION = "select "+ALL_GLOBAL_COLUMNS+" from " + GLOBAL_TABLE_PLACEHOLD + " where xid = ?";
+
+
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION_ID.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION_BY_ID = "select "+ALL_GLOBAL_COLUMNS+" from " + GLOBAL_TABLE_PLACEHOLD + " where transaction_id = ?";
+
+
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION_BY_STATUS.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION_BY_STATUS = "select "+ALL_GLOBAL_COLUMNS+" from " + GLOBAL_TABLE_PLACEHOLD +
+            " where status in (" + PRAMETER_PLACEHOLD + ") order by gmt_modified limit ?";
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION_FOR_RECOVERY_MYSQL.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION_FOR_RECOVERY_MYSQL = "select "+ALL_GLOBAL_COLUMNS+" from " + GLOBAL_TABLE_PLACEHOLD + " where status in (" +
+            "0, 2, 3, 4, 5, 6, 7, 8, 10 ,12, 14) order by gmt_modified limit ?";
+
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION_FOR_RECOVERY_ORACLE.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION_FOR_RECOVERY_ORACLE = "select A.* from ( select "+ALL_GLOBAL_COLUMNS+" from " + GLOBAL_TABLE_PLACEHOLD + " where status in (" +
+            "0, 2, 3, 4, 5, 6, 7, 8, 10 ,12, 14) order by gmt_modified ) A where ROWNUM <= ?" ;
```

Comment:
maybe we should  use status enum

Reply:
"0, 2, 3, 4, 5, 6, 7, 8, 10 ,12, 14"  is the code of  GlobalStatus enum